src/uwtools/rocoto.py

```diff
@@ -182,13 +182,47 @@ def _add_task_dependency(self, e: Element, config: dict) -> None:
         :param config: Configuration data for this element.
         """
         e = SubElement(e, STR.dependency)
-        for key, block in config.items():
-            tag, _ = self._tag_name(key)
```

Comment:
I took this line out on all following methods, but reading a bit more into this, I think it would be useful to add this line back in on my methods.